src/main/java/TaskList.java

```diff
@@ -0,0 +1,196 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private List<Task> tasks;
+
+    /**
+     * Initialises the TaskList according to a list of tasks that have been saved
+     * @param savedTasks Tasks that have been saved represented as a List<List<String>>
+     */
+    public TaskList(List<List<String>> savedTasks) {
+        tasks = new ArrayList<>();
+        for (List<String> savedTask : savedTasks) {
+            String type = savedTask.get(0);
```

Comment:
this portion can be quite confusing for other developers, as there is no documentation as to what index 0 to 3 are. you could consider creating a SavedTask class or something that has type, description and whatnot. then, you just take in a `List<SavedTask>`.